![image](https://github.com/rltpwns95/Face_Recognition/assets/124419697/005b9a3c-dc52-4a90-93c8-6e8a3a4cf627)<br><br>
![image](https://github.com/rltpwns95/Face_Recognition/assets/124419697/e5bc3d24-24a1-4e58-a256-b5ba339a6d89)<br><br>
![image](https://github.com/rltpwns95/Face_Recognition/assets/124419697/a2ecd38a-a2aa-48eb-a1ca-f325bcfc33ee)<br>
mediapipes.py를 실행했을 때의 사진(mediapipe.py로 만들었을 때 문제가 발생하여 뒤에 s를 붙여줬다.)
